gists/helm-component-extraction/loki/cloudformation/Makefile

```diff
@@ -0,0 +1,72 @@
+######################################################################
```

Comment:
Do we need cloudformation in this instance? If we are not using it to upgrade existing loki its a bit confusing that it is included.